karma.conf.js

```diff
@@ -13,8 +13,10 @@ module.exports = function(config) {
       'bower_components/angular-mocks/angular-mocks.js',
       'bower_components/angular-route/angular-route.js',
       'bower_components/angular-resource/angular-resource.js',
+	  'bower_components/angular-ui-router/release/angular-ui-router.js',
```

Comment:
@rohitnarula7176 you missed this...if you carefully look, you will notice its a tab in the front of the line instead of a space